### PR TITLE
Example screenshots

### PR DESCRIPTION
**Pull request by njooma**
Eliot requested screenshots: 

> https://github.com/viamrobotics/viam-flutter-sdk/tree/main/example/viam_example_app
> it would be really nice to put some screenshots of what the sample apps look like

Which got me doing a bit more documentation :)

### CODE REVIEW

**Review by clintpurser — APPROVED** [review]
LGTM! One more  non-blocking question, why use mac screenshots as opposed to mobile? do we fully support mac builds?

File: example/viam_example_app/README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Viam Example Flutter App
 
-This example app uses Viam's Flutter SDK to connect to and control a robot. It was created using VSCode's Flutter plugin and shows the basic use cases for how the Viam Flutter SDK can be used to connect manage your fleet of Smart Machines.
+This example app uses Viam's Flutter SDK to connect to and control a robot. It was created using VSCode's Flutter plugin and shows the basic use cases for how the Viam Flutter SDK can be used to connect manage your fleet of robots.
```
[the remark this answers is not in the document]
Comment (njooma) [reply example/viam_example_app/README.md:3]: [From @cheukt](https://github.com/viamrobotics/viam-python-sdk/pull/457#discussion_r1355813394)

> per our second-to-last all hands, I think the direction has been to not particularly change things up in documentation/code - given the rest of our code still uses robot, I think using robot makes sense until we're told to switch

**Comment by njooma** [comment]
Used macOS because our SDK supports it. The auth0_flutter SDK sadly doesn't support desktop, but because this example uses API keys, it's fully desktop compatible!

**Comment by njooma** [comment]
Also because my simulator wasn't working on my computer for some reason that day 🤷‍♂️ and I didn't feel like restarting my computer then

**Pull request merged**